xds/internal/xdsclient/xdsresource/type_cds.go

```diff
@@ -85,4 +85,7 @@ type ClusterUpdate struct {
 
 	// Raw is the resource from the xds response.
 	Raw *anypb.Any
+	// All the string valued metadata of filter_metadata type
+	// "com.google.csm.telemetry_labels".
+	StringMD map[string]string
```

Comment:
Bikeshedding: How about `TelemetryLabels`?

Reply:
Sure. That's what the scaled up LB Policy configs use as the field name, so that makes sense.